de.metas.adempiere.adempiere/base/src/main/java/org/adempiere/ad/window/api/impl/ADWindowDAO.java

```diff
@@ -394,12 +395,14 @@ public void copyWindow(@NonNull final AdWindowId targetWindowId, @NonNull final
 		logger.debug("Copying from: {} to: {}", sourceWindow, targetWindow);
 
 		copy()
-				.setSkipCalculatedColumns(true)
-				.addTargetColumnNameToSkip(I_AD_Window.COLUMNNAME_Name)
-				.addTargetColumnNameToSkip(I_AD_Window.COLUMNNAME_InternalName)
-				.setFrom(sourceWindow)
-				.setTo(targetWindow)
-				.copy();
+		.setSkipCalculatedColumns(true)
+		// skip it because other wise the MV will fill the name from the AD_Element of the original window and unique name constraint will be broken
+		.addTargetColumnNameToSkip(I_AD_Window.COLUMNNAME_AD_Element_ID)
+		.addTargetColumnNameToSkip(I_AD_Window.COLUMNNAME_Name)
+		.addTargetColumnNameToSkip(I_AD_Window.COLUMNNAME_InternalName)
```

Comment:
i tihnk u can also skip Description and Help because those are also coming from AD_Element...